### DISCUSSION TITLE
22000.469 explorer won't start with version from latest commit

### DISCUSSION THREAD

**Discussion by f242** [valinet/ExplorerPatcher#729] [General]
just blank screen with cursor, i boot from live usb to delete .dll

**Comment by abcpop** [comment]
I confirm black screen + cursor  with  Win 11 22000.469 and  EP 22000.466.41.15 
Not possible to launch explorer from task manager
Rolled back to previous restore point through sysdm.cpl

**Comment by abcpop** [comment]
Jumped to EP 22000.469.42.0 
All fine with Win 11 22000.469 
Many Thanks

**Comment by Dittotoo** [comment]
Blank screen here as well utilizing weather widget (Win 11 22000.469 / EP 22000.469.42.0)
Reverted back to EP 22000.469.41.16.

**Reply by valinet** [reply]
Have you tried running `explorer` manually after the update? Also, before updating, enable the console in Properties - Advanced so you can see what crashes and when. For an eventual crash, can you enable full crash dumps (as described [here](https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps)) and attempt to collect some and upload them here? It's a pre-release, first attempt at this, there may be some issues, but I rather have them fixed than ignored. Thanks.
